qemu/tests/block_copy.py

```diff
@@ -416,3 +418,16 @@ def verify_md5(self, file_name):
         if status != 0:
             raise error.TestFail("File %s changed, md5sum check output: %s" %
                                  (file_name, output))
+
+    def reopen(self, reopen_image):
+        """
+        Closing the vm and reboot it with the backup image.
+        :param reopen_image: the image that vm reopen with.
+        """
+        vm = self.vm
+        vm.destroy()
+        self.params["image_name_%s" % self.tag] = reopen_image
+        vm.create(params=self.params)
+        self.vm = vm
```

Comment:
I think it is better to use following code to recreate vm.
`self.vm.destroy()
self.vm.create(params=xxx)`